src/algebra/gcd_monoid/div.lean

```diff
@@ -31,7 +31,7 @@ namespace nat
 
 /-- Given a nonempty finset `s` and a function `f` from `s` to `ℕ`, if `d = s.gcd`,
 then the `gcd` of `(f i) / d` is equal to `1`. -/
-theorem coprime_of_div_gcd {β : Type*} {f : β → ℕ} (s : finset β) {x : β} (hx : x ∈ s)
+theorem gcd_div_eq_one {β : Type*} {f : β → ℕ} (s : finset β) {x : β} (hx : x ∈ s)
```

Comment:
Omitting the second `gcd` feels acceptable to me. So let's merge this if @Vierkantor is also happy.